src/elementary_operators.jl

```diff
@@ -191,6 +192,144 @@ function identity_operator(s::IndsNetworkMap; kwargs...)
   return ITensorNetwork(Op("I"), operator_inds)
 end
 
+" Create an operator bitstring corresponding to the number x"
+function point_to_opsum(s::IndsNetworkMap, x::Number, dim::Int)
+  ttn_op = OpSum()
+  ind_to_ind_value_map = calculate_ind_values(s, x, dim)
+  string_sites = [
+    (ind_to_ind_value_map[only(s[v])] == 1) ? ("Dup", v) : ("Ddn", v) for
+    v in dimension_vertices(s, dim)
+  ]
+  add!(ttn_op, 1.0, (string_sites...)...)
+  return ttn_op
+end
+
+" Function to manipulate delta functions into an operator. Defaults to map_to_zero behavior"
+function delta_kernel(
+  s::IndsNetworkMap,
+  points::Vector{<:Vector},
+  dimsl::Vector{<:Vector}=[[i for i in 1:length(xs)] for xs in points];
+  remove_overlap=true,
+  coeff=nothing,
+  include_identity=true,
+  promote_operator=true,
+  truncate_kwargs...,
+)
+  if isnothing(coeff)
+    coeff = -1 # assume we are making map_to_zero
+  end
+
+  ψ = coeff * delta_xyz(s, points, dimsl; truncate_kwargs...)
+
+  if include_identity
+    ψ = const_itn(s) + ψ
+  end
+
+  if remove_overlap && length(points) > 1
```

Comment:
So does this work in arbitrary dimensions? E.g. in `3D`, mapping the `6` faces of the unit cube to `0`?

Reply:
Yup, see the tests (although I haven't tested more than 3D)

Reply:
So the output bond dimension scales, at worst, quadratically in the number of points?

Reply:
Yeah something like that, for every plane/line/point you want you might need to remove the intersections. But the "upper triangular" sort of quadratic scaling and not full quadratic (`p*(p-1)/2`)